github-actions/bazel/configure-remote/setup_remote_exec.sh

```diff
@@ -1,7 +1,8 @@
+build --config=remote-cache`:u+=`
+build --config=remote`,await r.default.promises.writeFile(e,u,"utf8")}}async function D(e){try{return await r.default.promises.readFile(e,"utf8")}catch{return""}}P(process.env.BAZELRC_PATH).catch(e=>{console.error(e),process.exitCode=1})});M();})();
```

Comment:
Do you re-deploy a new orb version? we should do that:

1. Because sometimes this shell script is too long and doesn't work (especially on Windows shell limits)
2. If we don't do the update now, we end up with weird situations like in the past e.g. where we the service key had to be rotated for security reasons- and the updates ended up being extremely frustrating

Reply:
In general, we should not put NodeJS scripts in Shell in github actions because the whole brittle shell mechanism only existed for CircleCI

Reply:
I did not deploy a new orb version as I was waiting until it was merged.

I refactored things around to no longer use the a shell script for the github action.